src/auro-radio-group.js

```diff
@@ -23,9 +30,12 @@ import styleCss from "./auro-radio-group-css.js";
  * @attr {Boolean} noValidate - If set, disables auto-validation on blur.
  * @attr {Boolean} required - Populates the `required` attribute on the element. Used for client-side validation.
  * @csspart radio-group - Apply css to the fieldset element in the shadow DOM
+ * @event auroRadioGroup-validated - Notifies that the `validity` value has changed.
+ * @event auroRadioGroup-helpText - Notifies that the `setCustomValidity` value has changed.
```

Comment:
Shouldn't these be removed?

Reply:
You must have been looking at a specific commit, these events are not present